components/layout/layout_task.rs

```diff
@@ -709,6 +710,39 @@ impl LayoutTask {
                                                        color,
                                                        ScrollPolicy::Scrollable));
             let origin = Rect(Point2D(Au(0), Au(0)), root_size);
+
+            // Print the display list.
+            fn print_items(list: &Box<DisplayList>, indentation: String) {
+                let items = list.all_display_items();
+                for item in items.iter() {
+                    match *item {
+                        DisplayItem::SolidColorClass(ref solid_color) => println!("{} SolidColor.", indentation),
+                        DisplayItem::TextClass(ref text) => println!("{} TextClass.", indentation),
+                        DisplayItem::ImageClass(ref image_item) => println!("{} ImageClass.", indentation),
+                        DisplayItem::BorderClass(ref image_item) => println!("{} BorderClass.", indentation),
+                        DisplayItem::GradientClass(ref image_item) => println!("{} GradientClass.", indentation),
+                        DisplayItem::LineClass(ref line_item) => println!("{} LineClass.", indentation),
+                        DisplayItem::BoxShadowClass(ref box_shadow_item) => println!("{} BoxShadowClass.", indentation),
+                    }
+                }
+
+                if list.children.len() != 0 {
+                    println!("#### Children stacking contexts list length: {}", list.children.len());
+                    let mut tmp = indentation.clone();
+                    tmp.push_str(&indentation);
```

Comment:
(Also, won't nested display lists first have 4 hash marks, then 8, then 16, then 32, etc. instead of 4, 8, 12, 16, … as I would expect?)